Moblin/Various/Model.swift

```diff
@@ -1204,13 +1204,14 @@ final class Model: NSObject, ObservableObject {
         DispatchQueue.main.async {
             let camera = self.getRtmpStream(streamKey: streamKey)?.camera() ?? rtmpCamera(name: "Unknown")
             self.makeToast(title: "\(camera) disconnected")
-            guard let cameraId = self.getRtmpStream(streamKey: streamKey)?.id else {
+            guard let stream = self.getRtmpStream(streamKey: streamKey) else {
                 return
             }
-            self.media.removeRtmpCamera(cameraId: cameraId)
+            self.media.removeRtmpCamera(cameraId: stream.id)
             if self.database.debug!.enableRtmpAudio! {
-                self.media.removeRtmpAudio(cameraId: cameraId)
+                self.media.removeRtmpAudio(cameraId: stream.id)
             }
+            self.setMicFromSettings()
```

Comment:
Maybe move this into the if statement above? The case I'm thinking about is is the user want to use an external bluetooth mic with RTMP video. If that case the external mic should continue to be used after the RTMP stream stops.

Reply:
OK, I changed it. had to add an additional check